hack/build/print-workspace-status.sh

```diff
@@ -42,6 +42,13 @@ STABLE_DOCKER_TAG ${APP_VERSION:-${KUBE_GIT_VERSION/+/-}}
 STABLE_DOCKER_REGISTRY ${DOCKER_REGISTRY:-us.gcr.io/chris-love-operator-playground}
 STABLE_DOCKER_PUSH_REGISTRY ${DOCKER_PUSH_REGISTRY:-${DOCKER_REGISTRY:-us.gcr.io/chris-love-operator-playground}}
 STABLE_IMAGE_REPOSITORY ${DOCKER_IMAGE_REPOSITORY:-cockroach-operator}
+
+RH_BUNDLE_REGISTRY ${RH_BUNDLE_REGISTRY:-quay.io/alinalion}
+RH_BUNDLE_IMAGE_REPOSITORY ${RH_BUNDLE_IMAGE_REPOSITORY:-cockroach-operator-bundle}
+RH_BUNDLE_VERSION ${RH_BUNDLE_VERSION:-1.0.2}
+RH_DEPLOY_PATH ${RH_DEPLOY_PATH:-deploy/certified-metadata-bundle/cockroach-operator}
+RH_BUNDLE_IMAGE_TAG ${RH_BUNDLE_IMAGE_TAG:-${RH_BUNDLE_VERSION:-1.0.2}}
```

Comment:
Let's not set a default version.  Would rather test that we have the version set and not accidentally set the wrong version

Reply:
done